rustsec/Cargo.toml

```diff
@@ -44,6 +45,7 @@ fix = ["cargo-edit"]
 git = ["crates-index", "git2", "home", "humantime", "humantime-serde"]
 dependency-tree = ["cargo-lock/dependency-tree"]
 vendored-openssl = ["git2/vendored-openssl"]
+osv-I-know-this-is-unstable = ["git", "chrono"]
```

Comment:
Perhaps simply `osv` suffices here, and you can leave a comment regarding stability?

Reply:
The format itself is not stabilized. I explicitly wanted to mark anything exporting to not-yet-finalized format as unstable, because it's not really exporting to OSV, it's exporting to a pre-release of OSV.

Also, renaming it to `osv` later will make it clear if e.g. some dependent code has been updated to the stable version or not. Not just the stable version of the code, but also the format.